src/index.html

```diff
@@ -1,14 +1,167 @@
+  achievement for all peoples and all nations, to the end that every individual and
+  every organ of society, keeping this Declaration constantly in mind, shall strive by
+  teaching and education to promote respect for these rights and freedoms and by
+  progressive measures, national and international, to secure their universal and
+  effective recognition and observance, both among the peoples of Member States
+  themselves and among the peoples of territories under their jurisdiction.
+    </p>
+    <h2>Article I</h2>
+    <p>All human beings are born free and equal in dignity and rights. They are
+  endowed with reason and conscience and should act towards one another in a
+  spirit of brotherhood.
+    </p>
+    <h2>Article 2</h2>
+    <p>Everyone is entitled to all the rights and freedoms set forth in this Declaration,
+  without distinction of any kind, such as race, colour, sex, language, religion,
+  political or other opinion, national or social origin, property, birth or other status.
+  Furthermore, no distinction shall be made on the basis of the political,
+  jurisdictional or international status of the country or territory to which a person
+  belongs, whether it be independent, trust, non-self-governing or under any other
+  limitation of sovereignty.
+    </p>
+    <h2>Article 3</h2>
+    <p>Everyone has the right to life, liberty and the security of person.</p>
+    <h2>Article 4</h2>
+    <p>No one shall be held in slavery or servitude; slavery and the slave trade shall be
+  prohibited in all their forms.
+    </p>
+    <h2>Article 5</h2>
+    <p>No one shall be subjected to torture or to cruel, inhuman or degrading treatment
+  or punishment.
+    </p>
+    <h2>Article 6</h2>
+    <p>Everyone has the right to recognition everywhere as a person before the law.</p>
+    <h2>Article 7</h2>
+    <p>All are equal before the law and are entitled without any discrimination to equal
+  protection of the law. All are entitled to equal protection against any
+  discrimination in violation of this Declaration and against any incitement to such
+  discrimination.
+    </p>
+    <h2>Article 8</h2>
+    <p>Everyone has the right to an effective remedy by the competent national tribunals
+  for acts violating the fundamental rights granted him by the constitution or by law.
+    </p>
+    <h2>Article 9</h2>
+    <p>No one shall be subjected to arbitrary arrest, detention or exile.</p>
+    <h2>Article 10</h2>
+    <p>Everyone is entitled in full equality to a fair and public hearing by an independent
+  and impartial tribunal, in the determination of his rights and obligations and of any
+  criminal charge against him.
+    </p>
+    <h2>Article 11</h2>
+    <p>
+      <ol>
+        <li>Everyone charged with a penal offence has the right to be presumed
+    innocent until proved guilty according to law in a public trial at which he
+    has had all the guarantees necessary for his defence.
+        </li>
```

Comment:
```suggestion
        <li>
          Everyone charged with a penal offence has the right to be presumed
          innocent until proved guilty according to law in a public trial at which he
          has had all the guarantees necessary for his defence.
        </li>
```
In `ol` and `li` tags also should be proper indentations